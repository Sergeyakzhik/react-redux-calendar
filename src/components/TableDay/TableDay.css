.table-day td {
  height: 2rem;
  text-align: left;
  margin: 0;
  width: 90%;
  position: relative;
}

.table-day td h4 {
  margin-top: 0rem;
  margin-bottom: 2rem;
  margin-left: 0.2rem;
}

.table-day tr td:first-child {
  width: 10%;
}

.table-day {
  width: 100%;
  border-spacing: 0.15rem
}

.table-day {
  width: 100%;
}

.table-day .event {
  margin: 0;
  height: 100%;
  position: absolute;
  width: 100%;
  top: 0;
  border: 0.08rem solid black;
}

.table-day .event h4 {
  margin: 0;
  font-size: 0.8rem;
}

.table-day .events-skeleton {
  position: absolute;
  width: 89.4%;
  height: 96.9%;
  top: 2.9rem;
  left: 7.2rem;
}

.table-day .events-skeleton table {
  table-layout: fixed;
  height: 100%;
  width: 100%;
}

.table-day .events-skeleton td {
  border: none;
  background-color: transparent;
  height: 15px;
}

.table-day .events-skeleton tbody td {
  text-align: center;
}
