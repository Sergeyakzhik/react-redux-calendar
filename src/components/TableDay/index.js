import React from 'react';
import './style.css';

let TableDay = props => (
  <div>
    <h2>{props.date}</h2>
    <table className="table-day">
      <tbody>
        {props.tableRows}
      </tbody>
    </table>
  </div>
);

export default TableDay;
