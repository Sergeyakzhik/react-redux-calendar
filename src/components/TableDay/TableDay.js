import React from 'react';
import './TableDay.css';

let TableDay = props => {
  return (
    <div>
      <h2>{props.date}</h2>
      <table className="table-day">
        {props.tableRows}
      </table>
    </div>
  );
}

export default TableDay;
