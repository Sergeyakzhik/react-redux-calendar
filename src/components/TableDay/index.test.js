import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'

import TableDay from './index';

Enzyme.configure({ adapter: new Adapter() })

describe('TableDay', () => {
  let props = {
    tableRows: [],
    date: 'date'
  }

  let component = shallow(<TableDay {...props} />);

  it('renders TableDay component', () => {
    expect(component).toMatchSnapshot();
  })

  it('checks TableDay date', () => {
    expect(component.find('h2').text()).toEqual('date');
  })
})
