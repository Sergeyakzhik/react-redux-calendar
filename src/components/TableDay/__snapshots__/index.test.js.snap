// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableDay render TableDay component 1`] = `
<div>
  <h2>
    date
  </h2>
  <table
    className="table-day"
  >
    <tbody />
  </table>
</div>
`;
